.travis.yml

```diff
@@ -1,83 +0,0 @@
-language: generic
```

Comment:
I might be missing something.. why are we removing this file?

Reply:
my mistake! that was a missed file merge.